selfdrive/car/tests/test_docs.py

```diff
@@ -74,8 +74,8 @@ def test_harnesses(self):
         if car.name == "comma body":
           raise unittest.SkipTest
 
-        self.assertNotIn(car.harness_kit.connector, [None, Harness.none], f"Need to specify car harness: {car.name}")
-        self.assertTrue(car.harness_kit.parts, f"Need to specify harness parts: {car.name} with {car.harness_kit.connector.value} connector")
+        self.assertTrue(car.car_parts.parts, f"Need to specify car parts: {car.name}")
```

Comment:
```suggestion
        self.assertTrue(len(car.car_parts.parts) > 0, f"Need to specify car parts: {car.name}")
```

i like being more clear here